.github/workflows/python-ci.yml

```diff
@@ -0,0 +1,181 @@
+name: Python CI
+on:
+  pull_request_target:
+    paths:
+      - python/**
+      - packages/sdk-codegen*/**
+
+  push:
+    branches:
+      - $default-branch
+    paths:
+      - python/**
+      - packages/sdk-codegen*/**
+
+env:
+  LOOKERSDK_BASE_URL: https://localhost:20000
+  LOOKERSDK_VERIFY_SSL: false
+  TOX_JUNIT_OUTPUT_DIR: results
+
+defaults:
+  run:
+    shell: bash
+    working-directory: python/
+
+jobs:
+  unit:
+    name: Unit - ${{ matrix.os }} / py${{ matrix.python-version }}
+    env:
+      TOX_JUNIT_OUTPUT_NAME: ${{ matrix.os }}.py${{ matrix.python-version }}
+    runs-on: ${{ matrix.os }}-latest
+    strategy:
+      matrix:
+        os:
+          - ubuntu
+          - macos
+          - windows
+        python-version:
+          - '3.6'
+          - '3.7'
+          - '3.8'
+          - '3.9'
+
+    steps:
+      - name: Repo Checkout
+        uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install Tox and any other packages
+        run: |
+          python -m pip install --upgrade pip
+          pip install tox
+      - name: Set LOOKERSDK_CLIENT_{ID,SECRET}
+        run: |
+          echo "LOOKERSDK_CLIENT_ID=${{ secrets[format('LOOKERSDK_CLIENT_ID__{0}', matrix.looker)]}}" >> $GITHUB_ENV
```

Comment:
Each Looker image has different ID/SECRET creds - this code retrieves them programmatically from secrets.
In order for this workflow to function we'll need someone who can see the "Settings" tab to add these secrets:
`LOOKERSDK_CLIENT_ID__7_20`
`LOOKERSDK_CLIENT_SECRET__7_20`
`LOOKERSDK_CLIENT_ID__7_18`
`LOOKERSDK_CLIENT_SECRET__7_18`
`GCP_PROJECT_ID`
`GCP_AR_READER_SA_KEY`

I'll create an internal doc describing how to get the values. Happy to do it myself if someone can give me the access

Reply:
Who can give you the access you're needing?

Reply:
all set. I've started a document that describes how these secrets are managed.